### PR TITLE
Upgrade dependencies to match quality checklist 

### LINKED ISSUE #28: Code Exchange quality checklist
The purpose of this checklist is to assess the state of this repository, and bring it to an acceptable level for long-term support by completing the tasks listed below.

**This checklist is used internally but can be used as a guidance for others**

> Please check every part of this checklist for us to process your request. You can do this by marking them with [x] instead of [ ].

### Documentation

- [x] `LICENSE` file exists in repository

- [x] [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md) exists in repository

- [x] [CONTRIBUTING](https://github.com/twilio-labs/.github/blob/master/CONTRIBUTING.md) exists in repository 

- [x] No broken links in README.md

- [x] Build status exists at top of README.md

- [x] Set up steps are complete and accurate

- [x] CodeExchange repository is linked in README.md

### GitHub

- [x] Dependabot set up

- [x] Dependabot automerge set up

- [x] Uses GitHub Actions (required for Twilio sample) or other CI

### Code

- [x] Packages/dependencies are up to date

- [x] Code compiles against latest packages/dependencies

- [x] Matches [CodeExchange Guidelines](https://github.com/twilio-labs/code-exchange/tree/master/guidelines) to the best of ability

### Tests

- [x] Functionality tests have been created

- [x] All tests pass

### App

- [x] Confirmed sample behaves as expected (after dependencies have been updated)

- [x] This file has been copied into a wiki entry for the respective repository or added as a GitHub issue

### General

- [x] App is released under a permissive license like MIT or Apache-2

- [x] I've read and followed the contribution guidelines

- [x] I've read and implemented my template according to the respective coding guidelines for my project

- [x] I agree and adhere to the Code of Conduct

- [x] The template has been authored by me / or my organization and will be kept up-to-date. If the template will not longer be maintained, I'll notify this project.

### PR DESCRIPTION
**Pull request by rojasTob**
Resolves #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,63 +3,116 @@
...
 
-First you need to have [python](https://www.python-lang.org/), [virtualenv](https://virtualenv.pypa.io) and [pip](https://pip.pypa.io/) installed on your system.
+| .NET | Java | Ruby | PHP | Node |
+| :--- | :--- | :----- | :-- | :--- |
+| [Done](https://github.com/TwilioDevEd/block-spam-calls-csharp) | [Done](https://github.com/TwilioDevEd/block-spam-calls-java)  | [Done](https://github.com/TwilioDevEd/block-spam-calls-ruby)  | [Done](https://github.com/TwilioDevEd/block-spam-calls-php) | [Done](https://github.com/TwilioDevEd/block-spam-calls-node)  |
 
-To run the app locally, clone this repository and `cd` into its directory:
+## Set up
 
-1. First clone this repository and `cd` into its directory:
```
Comment (bharatmurali) [inline README.md:20]: fix numbering

File: README.md [hunk 2]
```diff
@@ -3,63 +3,116 @@
...
 
-Once you have started ngrok, update your TwiML app's voice URL setting to use your ngrok hostname, so it will look something like this:
+    ```bash
+    http://88b37ada.ngrok.io/
+    ```
 
-```
-http://88b37ada.ngrok.io/
-```
+That's it!
```
Comment (bharatmurali) [inline README.md:82]: we should provide more instructions on how this works and how to test it

Comment (bharatmurali) [reply README.md:82]: additionally, this isn't working for me. im getting this: 
```  File "/Users/bmurali/Developer/ce-apps/block-spam-calls-python/block_spam_calls/app.py", line 24, in block_spam_calls
    should_be_blocked = blocker_func(add_ons['results'].get(blocker_name, {}))
  File "/Users/bmurali/Developer/ce-apps/block-spam-calls-python/block_spam_calls/app.py", line 47, in should_be_blocked_by_ekata
    return ekata['result']['reputation_level'] >= 4
KeyError: 'reputation_level'```

i wonder if the add-on changed their API :(

File: README.md (hunk 1 above)
Comment (rojasTob) [reply README.md:20]: The enumeration works in this way too. However, I think is a good idea to mention the specific number on each instruction. Thanks!

File: README.md (hunk 2 above)
Comment (rojasTob) [reply README.md:82]: Hey Bharat, sorry about that! Could you please help me with this one? I've changed Whitepages to Ekata, but just changed the name of the add-on, then I executed the tests. Also, I've tested the POST method with a REST client (Postman). Could you help me with some steps to test the app behavior without a REST client? Thanks!

Comment (bharatmurali) [reply README.md:82]: the app seems to be working, so I think we are ok for now. I would add one more line to the final step staying "click on `Call` to test your app"

**Comment by rojasTob** [comment]
Hi @bharatmurali, changes are done. Thanks! :)

**Pull request merged**